bower.json

```diff
@@ -32,6 +32,7 @@
     "leaflet-tilelayer-geojson": "*",
     "Leaflet.utfgrid": "danzel/Leaflet.utfgrid",
     "Leaflet.awesome-markers": "*",
+    "Leaflet.vector-markers": "leaflet.vector-markers#0.0.3",
```

Comment:
Bower couldn't load the library when just using the version number, also this is what it saved into the bower.json when using ``bower install leaflet.vector-markers --save-dev``. 
But maybe this is a local problem of mine, when the pure version number is supposed to work also? I'll try.